src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawer {
+    private double area;
```

Comment:
please let's remove this field and its getter and setter

Reply:
add